proprietary/tokens/src/components/ams/tabs.tokens.json

```diff
@@ -2,11 +2,12 @@
   "ams": {
     "tabs": {
       "list": {
+        "background-color": { "value": "{ams.color.primary-white}" },
         "border-bottom": { "value": ".125rem solid {ams.color.primary-blue}" }
       },
       "button": {
         "background-color": { "value": "transparent" },
-        "border": { "value": "none" },
+        "border": { "value": "0" },
```

Comment:
These and the corresponding css rules aren't necessary anymore, right? These values are already set in the reset

Reply:
They would be helpful for teams theming our application who have borders on their tab buttons. The `0` value in our token then expresses that our Tab Buttons don’t have a border.

Reply:
We decided to focus on tokens we use ourselves for now.